src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,58 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie
+};
+
+export const MovieCard: React.FC<Props> = ({
+  movie: {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  },
```

Comment:
better destructure props inside your component, in future you may have more props and it'll be hard to manage them in function params